charmhelpers/core/hookenv.py

```diff
@@ -57,6 +57,17 @@
                  'This may not be compatible with software you are '
                  'running in your shell.')
 
+WL_STATE_ACTIVE = 'active'
+WL_STATE_BLOCKED = 'blocked'
+WL_STATE_MAINTENANCE = 'maintenance'
+WL_STATE_WAITING = 'waiting'
+
+WL_STATES = [
+    WL_STATE_ACTIVE,
+    WL_STATE_BLOCKED,
+    WL_STATE_MAINTENANCE,
+    WL_STATE_WAITING]
+
```

Comment:
As we are Python3 here, why not use an `Enum`: https://docs.python.org/3/library/enum.html

Reply:
Thanks for the tip, I've switched to an Enum and its much tidier